src/services/authService.js

```diff
@@ -20,15 +20,27 @@ const signIn = async (email, password) => {
     }
 }
 /**
- * Creates a new user within the firebase application with the given email and password
+ * Signs the user up to the firebase application.
+ * After they have successfully created their account, their first and last names
+ * are added to the firestore db
  * @param {*} email The email of the user as a string
  * @param {*} password The password of the user as a string
+ * @param {*} first_name The first name of the user as a string
+ * @param {*} last_name The last name of the user as a string
  */
-const signUp = async (email, password) => {
+const signUp = async (email, password, first_name, last_name) => {
     try{
-        await firebaseConnection
+        await firebaseApp
```

Comment:
should probs leave this as firebaseConnection

Reply:
good catch~~